src/packages/utils/webpack.prod.config.js

```diff
@@ -20,13 +20,19 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.mjs$/,
+        include: /node_modules/,
```

Comment:
we shouldn't be including `node_modules`, also `mjs` files don't need to be transpiled

Reply:
removing this fails build process with error:

```
ERROR in ../../../node_modules/jotai/esm/index.mjs 805:18-29
export 'useCallback' (imported as 'useCallback') was not found in 'react' (possible exports: __esModule)
 @ ../../data/library.ts
 @ ../utils.ts 258:0-52 258:0-52
 @ ./index.js 1:0-72 1:0-72 1:0-72 1:0-72
```

must be I'm doing something wrong,

```
node --version # v18.17.1
cd excalidraw/src/packages/utils
yarn build:umd
# webpack 5.76.0 compiled with 13 errors and 483 warnings in 20460 ms
```